src/mappings/generatedColumns.ts

```diff
@@ -66,13 +66,33 @@ export const extractClaimInfo = (args: any[]) => {
     args?.[1]?.value?.claim || {}
   )[0];
 
+  let cddId;
+  let jurisdiction;
+  if (claimType === "CustomerDueDiligence") {
+    cddId = JSONStringifyExceptStringAndNull(
+      args?.[1]?.value?.claim?.CustomerDueDiligence
+    );
+  } else if (claimType === "Jurisdiction") {
```

Comment:
(nit) export the enum from the mapper after all and use it here?

Probably better to use the enum over a string and it should get rid of the Github lint warning.